### DISCUSSION TITLE
Enable "donate custom amount" features by default

### DISCUSSION THREAD

**Discussion by sethmlarson** [community/community#5005] [Other Feature Feedback, Questions, & Ideas] [answered]
By default both the "one-time custom amount" and "monthly custom amount" are unticked which means most sponsors only accept the minimum tier that they've customized. This results in many sponsors not accepting anything below $5 as I've only seen a few people that have a minimum tier of $1.

By changing these options to be on by default lots of good things happen:

- Countries which have unfavorable conversion rates between the donator currency and USD for example can support people they want more freely without donating more than they're comfortable with.
- People who want to sponsor a certain amount that doesn't correspond with a tier can sponsor for any amount they choose and get the benefits of whatever tier is less than or equal to their sponsorship amount. Means more money for recipients!
- I wasn't even aware you couldn't donate custom amounts by default until I received a message from someone about it and they promptly sponsored me after I enabled it. How many other cases like this exist where the other person doesn't reach out? Probably missing out on a lot of sponsors!

An alternate suggestion is to alert the user that their custom amounts are disabled meaning the minimum sponsorship they'll accept is X where X is the least expensive sponsorship tier they've configured.

**Comment by cheshire137** [comment, marked as answer]
Hi there! I'm a dev on the Sponsors team, and we agree with you. We're enabling custom amounts by default on October 3, 2022 -- see https://github.blog/changelog/2022-09-13-github-sponsors-change-notice-for-the-custom-amount-setting/.

**Comment by Hexstream** [comment]
Ideally, there should be a way to preemptively opt-out of custom tiers before October 3.

I simply DO NOT need custom tiers given [my extensive tier system](https://sponsors.hexstreamsoft.com/about/#tiers), and it would uglify [my donations log](https://sponsors.hexstreamsoft.com/log/#donations)...

**Comment by Hexstream** [comment]
The announcements were not clear at all, I thought that Custom Amounts would be forcibly enabled on 3 october 2022 (today) but that we could just disable it after that, since it was said that they would be enabled "_by default_".

In fact, **it is now impossible to disable Custom Amounts**, which to me is completely ridiculous.

I set the Minimum Custom Amount to my biggest tier (10000$) as a workaround,
but this is just plain ridiculous, and gratuitously uglifies the UI for everyone.

**Please make it possible to disable custom amounts again!!**
